src/module.js

```diff
@@ -918,18 +956,47 @@ export default class DailyIframe extends EventEmitter {
           resolve({ workerId: msg.workerId });
         }
       };
-      this._sendIframeMsg({ action: DAILY_METHOD_SET_NETWORK_TOPOLOGY,
+      this.sendMessageToCallMachine({ action: DAILY_METHOD_SET_NETWORK_TOPOLOGY,
                             opts }, k);
     });
   }
 
   setPlayNewParticipantSound(arg) {
+    methodNotSupportedInReactNative();
     if (!(typeof arg === 'number' ||
           arg === true ||
           arg === false)) {
       throw new Error(`argument to setShouldPlayNewParticipantSound should be true, false, or a number, but is ${arg}`);
     }
-    this._sendIframeMsg({ action: DAILY_METHOD_SET_PLAY_DING, arg });
+    this.sendMessageToCallMachine({ action: DAILY_METHOD_SET_PLAY_DING, arg });
+  }
+
+  on(eventName, k) {
```

Comment:
these methods were simply moved up from the bottom of the file, so that they were grouped under the comment header "instance methods" rather than "internal methods"